runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -715,6 +741,47 @@ else if (activeTab != null
 			tagManager.removeTag(itemId, activeTab.getTag());
 			bankSearch.search(InputType.SEARCH, TAG_SEARCH + activeTab.getTag(), true);
 		}
+		else if (activeTab != null
+			&& event.getWidgetId() == WidgetInfo.BANK_ITEM_CONTAINER.getId()
+			&& event.getMenuAction() == MenuAction.RUNELITE
+			&& event.getMenuOption().startsWith(EDIT_QUANTITY))
+		{
+			// Add "edit-quantity" menu entry to all items in bank while tab is selected
+			event.consume();
+			final ItemComposition item = getItem(event.getActionParam());
+			final int itemId = item.getId();
+			final ItemComposition itemComposition = itemManager.getItemComposition(itemId);
+			final String name = itemComposition.getName();
+
+			final int initialValue = tagManager.getQuantityOfItemForTag(activeTab.getTag(), itemId, false);
+
+			chatboxPanelManager.openTextInput(name + " quantity:")
+				.addCharValidator(FILTERED_CHARS)
+				.value(String.valueOf(initialValue))
+				.onDone((newValue) ->
+					clientThread.invoke(() ->
+					{
+
+						// Set quantity
+						int amount;
```

Comment:
Maybe this should test if it's a valid number only and convert it to a stack with `QuantityFormatter:quantityToStackSize` here. Otherwise unset it.